cachito/web/models.py

```diff
@@ -231,6 +232,39 @@ def to_purl(self):
                     f"Unknown protocol in npm package version: {self.version}"
                 )
 
+        elif self.type == "pypi":
```

Comment:
Do we need this as part of this PR? This seems part of the ICM work @chmeliik is working on.

Reply:
I have marked this as WIP since I wanted to discuss couple of things regarding this today. I'll bring this up after the mtg.